doku.php

```diff
@@ -11,6 +11,16 @@
 // update message version - always use a string to avoid localized floats!
 $updateVersion = "50";
 
+// let people know if they are running an unsupported version of PHP
+$required_php = "5.6";
+
+if (version_compare(phpversion(), $required_php, '<')) {
+
+  die('<h2 style="margin:1.2em;text-align:center;color:red;line-height:1.6;" >Error: You are currently running PHP/'.phpversion().'.<br>
+       <span style="color:#606060;" >DokuWiki requires PHP/'.$required_php.' or higher.</span></h3>');
+
+} else {
```

Comment:
This else is not required as `die()` terminates the script.

Reply:
Hi mitchitux, is it ok, when I write in german? I kwow this is not usual here but it is quite arduous to me to write english...